libbindgen/src/ir/ty.rs

```diff
@@ -685,30 +685,40 @@ impl Type {
                         CXCursor_TemplateRef => {
                             let referenced = location.referenced().unwrap();
                             let referenced_ty = referenced.cur_type();
-                            let referenced_declaration =
-                                Some(referenced_ty.declaration());
+
+                            debug!("TemplateRef {:?} {:?} {:?}",
```

Comment:
Please tweak this formatting so that it is easier to read the logs:

```rust
debug!("TemplateRef: location = {:?}; referenced = {:?}; referenced_ty = {:?}", ...);
```